releng/org.eclipse.n4js.targetplatform/N4JS.setup

```diff
@@ -25,270 +26,273 @@
-    <repository
-        url="http://meysholdt.github.io/xtext_stateful_lexer/updatesite"/>
-  </setupTask>
-  <setupTask
-      xsi:type="setup.p2:P2Task"
-      label="SWTBot">
-    <requirement
-        name="org.eclipse.swtbot.ide.feature.group"/>
-    <repository
-        url="http://download.eclipse.org/technology/swtbot/releases/2.5.0"/>
-  </setupTask>
-  <setupTask
-      xsi:type="setup.p2:P2Task"
-      label="M2E">
-    <requirement
-        name="org.eclipse.m2e.feature.feature.group"/>
-    <repository
-        url="http://download.eclipse.org/technology/m2e/releases"/>
-  </setupTask>
-  <setupTask
+
+	<setupTask
+			xsi:type="setup:ResourceCreationTask"
+			excludedTriggers="STARTUP MANUAL"
+			content="&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?>&#xD;&#xA;&lt;section name=&quot;Workbench&quot;>&#xD;&#xA;&#x9;&lt;section name=&quot;org.eclipse.jdt.internal.ui.packageview.PackageExplorerPart&quot;>&#xD;&#xA;&#x9;&#x9;&lt;item value=&quot;true&quot; key=&quot;group_libraries&quot;/>&#xD;&#xA;&#x9;&#x9;&lt;item value=&quot;false&quot; key=&quot;linkWithEditor&quot;/>&#xD;&#xA;&#x9;&#x9;&lt;item value=&quot;2&quot; key=&quot;layout&quot;/>&#xD;&#xA;&#x9;&#x9;&lt;item value=&quot;2&quot; key=&quot;rootMode&quot;/>&#xD;&#xA;&#x9;&#x9;&lt;item value=&quot;&amp;lt;?xml version=&amp;quot;1.0&amp;quot; encoding=&amp;quot;UTF-8&amp;quot;?&amp;gt;&amp;#x0D;&amp;#x0A;&amp;lt;packageExplorer configured=&amp;quot;true&amp;quot; group_libraries=&amp;quot;1&amp;quot; layout=&amp;quot;2&amp;quot; linkWithEditor=&amp;quot;0&amp;quot; rootMode=&amp;quot;2&amp;quot; sortWorkingSets=&amp;quot;false&amp;quot; workingSetName=&amp;quot;&amp;quot;&amp;gt;&amp;#x0D;&amp;#x0A;&amp;lt;localWorkingSetManager&amp;gt;&amp;#x0D;&amp;#x0A;&amp;lt;workingSet editPageId=&amp;quot;org.eclipse.jdt.internal.ui.OthersWorkingSet&amp;quot; factoryID=&amp;quot;org.eclipse.ui.internal.WorkingSetFactory&amp;quot; id=&amp;quot;1382792884467_1&amp;quot; label=&amp;quot;Other Projects&amp;quot; name=&amp;quot;Other Projects&amp;quot;/&amp;gt;&amp;#x0D;&amp;#x0A;&amp;lt;/localWorkingSetManager&amp;gt;&amp;#x0D;&amp;#x0A;&amp;lt;activeWorkingSet workingSetName=&amp;quot;Other Projects&amp;quot;/&amp;gt;&amp;#x0D;&amp;#x0A;&amp;lt;allWorkingSets workingSetName=&amp;quot;Other Projects&amp;quot;/&amp;gt;&amp;#x0D;&amp;#x0A;&amp;lt;/packageExplorer&amp;gt;&quot; key=&quot;memento&quot;/>&#xD;&#xA;&#x9;&lt;/section>&#xD;&#xA;&lt;/section>&#xD;&#xA;"
+			targetURL="${workspace.location|uri}/.metadata/.plugins/org.eclipse.jdt.ui/dialog_settings.xml"
+			encoding="UTF-8">
+		<description>Initialize JDT's package explorer to show working sets as its root objects</description>
+	</setupTask>
+	<setupTask
+			xsi:type="setup:VariableTask"
+			name="eclipse.target.platform"
+			value="None"
+			storageURI="scope://Workspace"/>
+	<setupTask
+			xsi:type="setup:VariableTask"
+			name="n4js.target.platform"
+			defaultValue="N4JS_Neon.2.IDE-2686"
```

Comment:
what about the suffix `.IDE-2686`?
iirc this is supposed to land on master; but should we change it to `.GH-...`?